### PR TITLE
docs(faq): add a question about autoplay

### PR DESCRIPTION
**Pull request by gkatsev**
A comment question, particularly around mobile devices.

### CODE REVIEW

**Comment by gkatsev** [comment]
@mister-ben would love your input on this.

File: docs/faq.md [hunk 1]
```diff
@@ -116,6 +118,38 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
...
+// or
+
+player.autoplay(true);
+```
+
+### Q: How can I autoplay a video on a mobile device?
+
+Most mobile devices have blocked autoplaying videos until recently.
+For mobile devices that don't support autoplaying, there's nothing that can be done.
+For those devices that support autoplaying, like iOS10, you must mute the video
```
Comment (mister-ben) [inline docs/faq.md]: "like iOS10 or Chrome 53+ on Android,"

File: docs/faq.md [hunk 2]
```diff
@@ -116,6 +118,38 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
...
+
+// or
+
+player.autoplay(true);
+```
+
+### Q: How can I autoplay a video on a mobile device?
+
+Most mobile devices have blocked autoplaying videos until recently.
+For mobile devices that don't support autoplaying, there's nothing that can be done.
```
Comment (mister-ben) [inline docs/faq.md]: ", autoplay is not supported by Video.js."

"nothing that can be done" invites digression into canvas hacks and the like.

**Comment by gkatsev** [comment]
Thanks @mister-ben, I've made updates.

**Pull request merged**